op-devstack/dsl/el.go

```diff
@@ -33,26 +33,67 @@ func (el *elNode) ChainID() eth.ChainID {
 }
 
 func (el *elNode) WaitForBlock() eth.BlockRef {
+	return el.waitForNextBlock(1)
+}
+
+func (el *elNode) WaitForFinalization() eth.BlockRef {
```

Comment:
This function is no longer used, I suggest we remove it. Finality of the chain is not needed for the tx inclusion testing. And we've existing DSL from CW & Anton that await finality.